packages/model/src/research-output.ts

```diff
@@ -128,6 +136,19 @@ export const researchOutputMapType = (
   return null;
 };
 
+export const researchOutputMapPublishingEntity = (
+  publishingEntity?: string | null,
+): ResearchOutputPublishingEntities | null => {
+  if (
+    publishingEntity &&
+    (publishingEntity === 'Team' || publishingEntity === 'Working Group')
+  ) {
+    return publishingEntity;
+  }
+
+  return null;
```

Comment:
This a case that should never happen. Either we throw because it shouldn't happen as we have migrated. Or we should default to Team and just have new entities correctly tagged.